docs/Frequently Asked Questions/Multiple Process Items/index.md

```diff
@@ -0,0 +1,98 @@
+
+### Why is this a problem?
+
+The issue is when there is code in a developer's custom `Application` class that assumes the content provider has already been initialized but an instance of that child `Application` class is created for a process outside the main process.
+
+For example:
+
+```
+public class MyApplication extends Application {
+
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        
+         ModuleUsingContentProviderForInit.doSomething();
+
+    }
+}
+
+```
+
+If an instance is created outside the main process, this application will crash with a runtime exception. 
+
+### Workaround
+
+Depending on the module that uses a content provider for initialization it could be possible to start/initialize it from the `onCreate` method of the extended `Application` class. It should be noted that the module would then need to be setup for a multiple process environment, which is not always the case.
+
+If the module can't be initialized in this way, then the `Application` child class will need to keep a flag that prevents code from executing that would cause the error. 
+
+For SDL the solution can be as follows:
+
+```
+public class MyApplication extends Application {
+
+    private static final String ROUTER_SERVICE_PROCESS = "com.smartdevicelink.router";
+    
+    boolean isSdlProcessFlag = false;
+    
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        
+        isSdlProcessFlag = isSdlProcess()
+        if (isSdlProcessFlag) {
+          //This application instance is running in the SDL process
+        	return;
+        }
+        
+        ModuleUsingContentProviderForInit.doSomething();
+
+    }
+    
+    /**
+    *
+    * @return if this process is the SDL router service process
+    */
+    private boolean isSdlProcess(){
+        int myPid = android.os.Process.myPid();
+        ActivityManager am = (ActivityManager)this.getSystemService(ACTIVITY_SERVICE);
+        if (am == null || am.getRunningAppProcesses() == null) {
+            return false;
+        }
+        
+        for (ActivityManager.RunningAppProcessInfo processInfo : am.getRunningAppProcesses()) {
+            if (processInfo != null && processInfo.pid == myPid) {
+                return ROUTER_SERVICE_PROCESS.equals(processInfo.processName);
+            }
+        }
+        return false;
+    }
+}
+
+```
+
+!!! NOTE
+If other callback methods in your `Application` class are used, they must also check this flag to prevent unintended behavior.
+!!!
+
+The use of this flag can help prevent errors when extending the `Application` class that assume it always has the main process started first. This solution could be modified to change the flag to monitor if this is the main process or not very easily. 
+
+
+### Custom `Application` classes instance for each process
```

Comment:
The code block in a header formats a little funny.  You may want to consider removing the ticks.

![Screen Shot 2019-10-03 at 10 54 32 AM](https://user-images.githubusercontent.com/22600496/66138217-3bbf6980-e5cc-11e9-961d-258937e6c69a.png)